OpenRA.Mods.Common/Traits/RevealOnDeath.cs

```diff
@@ -27,7 +27,7 @@ public class RevealOnDeathInfo : ConditionalTraitInfo
 		[Desc("Radius of the reveal around this actor.")]
 		public readonly WDist Radius = new WDist(1536);
 
-		[Desc("Can this actor be revealed through shroud generated by the GeneratesShroud trait?")]
+		[Desc("Can this actor be revealed through shroud generated by the CreatesShroud trait?")]
```

Comment:
`[Desc("Can this actor be revealed through shroud generated by the " + nameof(CreatesShroud) + "trait?")]` would provide compiler guarantees that this error can't happen again.

Reply:
Good idea. I found some more occasions to use it.